### ISSUE TITLE
typo in the recurrent page

### ISSUE THREAD

**Issue by tetsuyasu** [fchollet/keras-docs-ja#127]
https://keras.io/ja/layers/recurrent/

in the RNN section,
誤: - fit()を呼ぶときは，stateful=False を指定してください．
正: - fit()を呼ぶときは，shuffle=False を指定してください．

**Comment by masstomato** [comment]
Thank you for your comments. [Fix it](https://github.com/keras-team/keras-docs-ja/commit/b50b387fcd11ccde0ac363578fa921b7a2cec650).
